src/olympia/users/management/commands/createsuperuser.py

```diff
@@ -80,6 +90,23 @@ def handle(self, *args, **options):
                 'api-secret': apikey.secret
             }))
 
+        if options.get('ui_test', False):
+            # json object for variables file
+            variables = {}
+            variables['api'] = {}
+
+            # export to variables.json object
+            variables['api'].update({
+                'olympia.dev': dict([
+                    ('username', user.username,),
+                    ('jwt_issuer', apikey.key),
+                    ('jwt_secret', apikey.secret)
+                ])
+            })
```

Comment:
Right, so it needs to be the hostname for the UI tests to work. I don't know the env variables that we would use on circle ci for different stages. I would like to change the hostname based on the stages however.